src/ecmascript/spidermonkey/util.h

```diff
@@ -38,4 +39,32 @@ jsval_to_boolean(JSContext *ctx, JS::Value *vp)
 	return (int)r_vp.toBoolean();
 }
 
+
+/* Since SpiderMonkey 52 the Mutable Handle Object
+ * is different for String and Number and must be
+ * handled accordingly */
+unsigned char *
+jshandle_value_to_char_string(JSContext *ctx, JS::MutableHandleValue *obj)
+{
```

Comment:
This function also should be rewritten to return struct string *

Reply:
O.K., agreed. Fixed.